webapp/platform/client/src/client4.ts

```diff
@@ -2086,6 +2086,13 @@ export default class Client4 {
         );
     };
 
+    getSharedChannelRemoteInfo = (channelId: string) => {
```

Comment:
nit: I'd use the plural here to signify that the response may return multiple remoteInfos
```suggestion
    getSharedChannelRemoteInfos = (channelId: string) => {
```